src/main/java/io/zold/api/RtTransaction.java

```diff
@@ -155,43 +186,34 @@ public long amount() throws IOException {
 
     @Override
     public String prefix() throws IOException {
-        try {
-            final String prefix = new ItemAt<>(
-                new SplitText(
-                    new TextOf(
-                        this.transaction
-                    ),
-                    new TextOf(";")
-                ).iterator(),
-                //@checkstyle MagicNumberCheck (1 line)
-                3
-            ).value().asString();
-            //@checkstyle MagicNumberCheck (1 line)
-            if (prefix.length() < 8 || prefix.length() > 32) {
-                throw new IllegalArgumentException("Invalid prefix size");
-            }
-            if (!RtTransaction.TRANSACTION.matcher(prefix).matches()) {
-                throw new IllegalArgumentException("Invalid base64 prefix");
-            }
-            return prefix;
-            //@checkstyle IllegalCatchCheck (1 line)
-        } catch (final Exception exception) {
-            throw new IOException(
-                "Invalid prefix string",
-                exception
-            );
+        final String prefix = new UncheckedText(
+            new IoCheckedScalar<String>(
+                () -> new ItemAt<Text>(
```

Comment:
@paulodamaso here too, we can avoid the lambda here and re-wrapping `String` by changing `transaction` to `IoCheckedScalar`.